framework/log/Target.php

```diff
@@ -266,6 +266,10 @@ public function getMessagePrefix($message)
             return call_user_func($this->prefix, $message);
         }
 
+        if (Yii::$app === null) {
+            return "[n/a]";
```

Comment:
I chose the value in brackes, because it looked so different otherwise. But I could change that. 

@qiangxue, @cebe please let me know if you would like me to update this PR - or if you have something completely different in mind.